jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/RadioButtonTests.java

```diff
@@ -1,53 +1,66 @@
 package io.github.epam.material.tests.inputs;
 
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.tools.Timer;
 import io.github.epam.TestsInit;
+import org.hamcrest.Matchers;
+import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
 
-import static io.github.com.StaticSite.radioDefaultPage;
-import static io.github.com.StaticSite.radioButtonFrame;
-import static org.hamcrest.Matchers.containsString;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static io.github.com.pages.inputs.RadioButtonPage.*;
+import static io.github.com.MaterialNavigator.openSection;
+import static io.github.com.StaticSite.radioButtonPage;
 
 
 public class RadioButtonTests extends TestsInit {
+    static private final ArrayList<String> labels = new ArrayList<>(Arrays.asList("First", "Second", "Third", "Disabled"));
```

Comment:
define this variable via
static private final List<String> labels = Arrays.asList("First", "Second", "Third", "Disabled");
same for others

Reply:
done